api/filter/logging.go

```diff
@@ -42,16 +43,17 @@ func RequestLogHandler() gin.HandlerFunc {
 
 		param, _ := c.Get("requestBody")
 
-		switch param.(type) {
+		switch paramType := param.(type) {
 		case []byte:
 			param = string(param.([]byte))
+			fmt.Printf("type of param: %#v", paramType)
```

Comment:
change to `log.info`

Reply:
Sorry, the original log library does not support info, warn, etc.. levels, need to submit another PR to do this.

Reply:
I change to log first.